packages/ui/src/components/va-virtual-scroller/VaVirtualScroller.stories.ts

```diff
@@ -1,12 +1,285 @@
 import { defineComponent } from 'vue'
-import VaVirtualScroller from './VaVirtualScroller.demo.vue'
+import { within } from '@storybook/testing-library'
+import { expect } from '@storybook/jest'
+
+import { VaVirtualScroller } from './'
+import { VaBadge } from '../va-badge'
+import { VaButton } from '../va-button'
 
 export default {
   title: 'VaVirtualScroller',
   component: VaVirtualScroller,
+  tags: ['autodocs'],
+}
+
+const hugeArrayBase = new Array(1000)
+const hugeArray = hugeArrayBase.fill(null).map((_, index) => index)
+const hugeObjectsArray = hugeArrayBase.fill(null).map((el, index) => ({ value: index }))
+
+export const Default = () => ({
+  components: { VaVirtualScroller },
+  data: () => ({ hugeArray }),
+  template: `
+    <va-virtual-scroller
+      :items="hugeArray"
+      :wrapper-size="200"
+      v-slot="{item}"
+    >
+      <div>{{ item }}</div>
+    </va-virtual-scroller>
+  `,
+})
+
+Default.play = async ({ step }) => {
+  const scroller = document.querySelector('.va-virtual-scroller') as HTMLElement
+
+  await step('wrapper height is 200px', async () => {
```

Comment:
We can not check height in interaction tests - as that would be tested in visual tests in chromatic anyway.

That's applicable to all tests.

Reply:
We generally want to test only functionality that's relevant to the story. So in case of `disabled` that would be disabled-ness itself 🙃 etc.

Reply:
Hi, @asvae, thanks for reviewing my pr. After reading your comment, I am assuming story without play function are worth having because the story is taken in considerate in visual tests, is that correct?

When reading documents at storybook, I have only vague understanding of visual test. I know it's testing the visual feature of the component, I know it's going to run in chromatic very pr is made. But that's all.

Could you tell me how do you get to know the specific knowledge to make that call?

Reply:
@xiongmao86 
> I am assuming story without play function are worth having
Absolutely yes. Actually, that's majority of value they provide. Developers would check relevant stories manually while working on components. Visual tests and interaction tests - are mostly safeguards to prevent paranoia :).

I'll try to answer your question if I understood it correctly.

Right now we have visual and interaction tests in [chromatic](https://www.chromatic.com/build?appId=64d608ab34b67e621ab50cfd&number=303).

All stories are covered with visual tests and chromatic shows us diffs between latest stable version and current version. So that we can approve change. That's enabled for all stories currently.

Visual tests motivate us to make stories visual in a sense that when feature breaks - something visual will change. That would allow chromatic to inform us and likely this change won't pass review and be bounced back.

In case of virtual scroll container size - if feature breaks - container size would change and that would be visible in chromatic.

Another thing storybook does - is interaction tests. (that's not the same as visual test) These are pretty much short e2e tests and allow us to catch stuff that is not detectable by visual tests or set up the story in specific way (scroll to certain point to make component visible etc).

We want to have interaction tests only in critical places and in pretty basic form.

I.e. we have `color` feature and it's big feature and you'll have a big interaction test if you want to test it fully. But in stories we only want to test "color feature works at all", not delve into details. Details should be handled in unit tests or specialized stories that pretty much test composabled directly (I don't have an example of that sadly, just an idea for now 😬).

Hope that helps.

Reply:
I am not very sure what do you mean by testing disabled-ness, 

I have found 3 occurrence of `disabled` prop. If `disabled` is true: 
1. `useEvent('scroll', handleScroll, true)` is not called, 
2. items is not dynamically loaded
3. `isDisabledVirtualTable` is false
by testing disabled-ness do you mean by testing on the scroll event in point 1?

Reply:
I am not very sure what do you mean by testing disabled-ness, 

I have found 3 occurrence of `disabled` prop. If `disabled` is true: 
1. `useEvent('scroll', handleScroll, true)` is not called, 
2. items is not dynamically loaded
3. `isDisabledVirtualTable` is false
by testing disabled-ness do you mean by testing on the scroll event in point 1?

 I have checked that `disabled` scroller rolls in respond to mouse wheels just like the non disabled scroller, how do I test for that?